LocalSettings.php

```diff
@@ -3651,7 +3651,7 @@
 		'default' => true,
 	],
 
-	// T11025
+	// NearbyPages
```

Comment:
```suggestion
	// NearbyPages - T11025
```
Might be worth keeping the task mentioned here.